code/game/objects/items/food/snacks.dm

```diff
@@ -312,3 +312,73 @@ GLOBAL_LIST_INIT(safe_peanut_types, populate_safe_peanut_types())
 	qdel(sample)
 
 	. = ..()
+
+
+/obj/item/food/cornchips
+	name = "boritos corn chips"
+	desc = "Triangular corn chips. They do seem a bit bland but would probably go well with some kind of dipping sauce."
+	icon_state = "boritos"
+	trash_type = /obj/item/trash/boritos
+	bite_consumption = 2
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/cooking_oil = 2, /datum/reagent/consumable/salt = 3)
+	junkiness = 20
+	custom_price = PAYCHECK_ASSISTANT * 0.8  //we are filled to the brim with flavor
+	tastes = list("fried corn" = 1)
+	foodtypes = JUNKFOOD | FRIED
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/cornchips/MakeLeaveTrash()
+	if(trash_type)
+		AddElement(/datum/element/food_trash, trash_type, FOOD_TRASH_POPABLE)
+
+/obj/item/food/cornchips/blue
+	name = "coolest ranch boritos corn chips"
+	desc = "Triangular corn chips. They do seem a bit bland but would probably go well with some kind of dipping sauce. This one is in coolest ranch!"
```

Comment:
the flavored variants should come with a different description, since they arent just bland corn chips with nothing on em.


also, coolest ranch allows for a dilbert 3 reference

Reply:
pass on the reference then

Reply:
yeah, do that. dilbert 3 reference optional